files/salt/admin/root/.inputrc

```diff
@@ -0,0 +1,4 @@
+"\C-[OA": history-search-backward
```

Comment:
Is this file was added by mistake? or what is the purpose?

Reply:
It was added by purpose because i think bash history is a cool thing.